bin/carthage.sh

```diff
@@ -10,26 +10,17 @@ set -euo pipefail
 xcconfig=$(mktemp /tmp/static.xcconfig.XXXXXX)
 trap 'rm -f "$xcconfig"' INT TERM HUP EXIT
 
+
 # For Xcode 12 make sure EXCLUDED_ARCHS is set to arm architectures otherwise
 # the build will fail on lipo due to duplicate architectures.
-# Xcode 12 Beta 3:
```

Comment:
These were no longer needed for the workaround and caused Alamofire to fail to build.